CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 librdkafka v2.8.0 is a maintenance release:
 
+* Extend Config Apis to support group config (#4939).
```

Comment:
Mapping should be done for AlterConfigs too otherwise there's a problem if they pass the `RD_KAFKA_RESOURCE_GROUP` there.